WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -133,7 +109,7 @@ private bool CheckForFlags(ApiResponseItem response)
 		bool shouldBan = false;
 
 		var coinflaglist = response.Cscore_section.Cscore_info;
-		shouldBan = coinflaglist.Any(cscoreInfo => _flagIds.Contains(cscoreInfo.Id));
+		shouldBan = coinflaglist.Any(cscoreInfo => WabiSabiConfig.RiskFlags?.Contains(cscoreInfo.Id) ?? false);
```

Comment:
This is where `is true` pattern matching shines in contrast to the old `== true`

```suggestion
		shouldBan = coinflaglist.Any(cscoreInfo => WabiSabiConfig.RiskFlags?.Contains(cscoreInfo.Id) is true);
```